### PR TITLE
Implement operations consist with code editor

### PR DESCRIPTION
**Pull request by imbyungjun**
This pull request must be merged before merging #394

### CODE REVIEW

**Comment by donggyu04** [comment]
Can one of the admins verify this patch?

File: ngrinder-controller/src/main/java/org/ngrinder/operation/cotroller/SystemConfigController.java [hunk 1]
```diff
@@ -88,10 +66,11 @@ public String save(@RequestParam final String content, ModelMap model) {
...
 	@RestAPI
-	@RequestMapping(value = "/api", method = RequestMethod.POST)
-	public HttpEntity<String> save(@RequestParam(required = true) final String content) {
+	@ResponseBody
+	@PostMapping(value = "/api")
+	public String save(@RequestParam String content) {
 		systemConfigService.save(checkNotEmpty(content, "content should be " +
 				"passed as parameter"));
-		return successJsonHttpEntity();
+		return "success";
```
Comment (songeunwoo) [inline ngrinder-controller/src/main/java/org/ngrinder/operation/cotroller/SystemConfigController.java]: ?

Comment (imbyungjun) [reply ngrinder-controller/src/main/java/org/ngrinder/operation/cotroller/SystemConfigController.java]: My bad. I'm gonna ensure backward compatibility

Comment (donggyu04) [reply ngrinder-controller/src/main/java/org/ngrinder/operation/cotroller/SystemConfigController.java]: how about
```
	public String save(@RequestParam String content) {
		systemConfigService.save(checkNotEmpty(content, "content should be " +
				"passed as parameter"));
		return returnSuccess();
``` 

returnSuccess() : https://github.com/naver/ngrinder/blob/develop/ngrinder-controller/src/main/java/org/ngrinder/common/controller/BaseController.java#L237

File: ngrinder-frontend/src/js/components/operation/ScriptConsole.vue [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            '- pluginManager (org.ngrinder.infra.plugin.PluginManager)\n' +
+            '- cacheManager (org.springframework.cache.CacheManager)\n' +
+            '\n' +
+            'Please type following and click the Submit button as a example\n' +
+            '\n' +
+            'print agentManager.getAllAttachedAgents()\n' +
+            '\n' +
+            'please refer nGrinder javadoc to find out more APIs on the given variables.\n';
+
+        cmOptions = {
```
Comment (donggyu04) [inline ngrinder-frontend/src/js/components/operation/ScriptConsole.vue]: Is there a reason to use computed in below code? 

https://github.com/naver/ngrinder/blob/445efe356db413d4e6c625fc2c3f36ad63607e62/ngrinder-frontend/src/js/components/common/CodeMirror.vue#L43

**Comment by imbyungjun** [comment]
This pr has been merged to `convert-to-vuejs-script` branch

**Pull request closed without merging**